harness/tests/experiment/pytorch/test_pytorch_lightning.py

```diff
@@ -87,3 +84,48 @@ def make_trial_controller_fn(
 
         with pytest.raises(AssertionError):
             utils.checkpointing_and_restoring_test(make_trial_controller_fn, tmp_path)
+
+    def test_lr_scheduler(self, tmp_path: pathlib.Path) -> None:
+        class OneVarLAFreq1(la_model.OneVarTrialLRScheduler):
+            def check_lr_value(self, batch_idx: int):
```

Comment:
q: when is this function called?

Reply:
the parent calls this as part of train_batch
https://github.com/determined-ai/determined/pull/2184/files#diff-1e0d3fc8c227931ea1220947977eb3378f3f12e3916faa15fa74b703d32c7f56R136